rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/TreeView.java

```diff
@@ -134,27 +136,24 @@ Canvas build() {
         this.tree = new Tree();
         tree.setModelType(TreeModelType.PARENT);
         tree.setNameProperty("name");
-        tree.setIdField("nodeId");
+        tree.setIdField(NODE_ID);
 
         this.treeGrid.setData(this.tree);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)